pkg/discovery/logsummary_test.go

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright the Sonobuoy contributors 2022
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package discovery
+
+import (
+	"testing"
+        "encoding/json"
+	"bytes"
+        "path/filepath"
+	"io/ioutil"
+)
+
+const (
+	//Name of the golden file that stores the expected result
+	LogSummaryGoldenFileName = "logsummary_test.golden"
```

Comment:
Same as other test comment; keep those constants just in the test itself. Too many test constants to define in both places.